astore/server/auth/auth.go

```diff
@@ -96,8 +101,12 @@ func (s *Server) Token(ctx context.Context, req *auth.TokenRequest) (*auth.Token
 		}
 
 		return &auth.TokenResponse{
-			Nonce: nonce[:],
-			Token: box.Seal(nil, []byte(token), &nonce, (*[32]byte)(clientPub), (*[32]byte)(s.serverPriv)),
+			Nonce:       nonce[:],
+			Token:       box.Seal(nil, []byte(token), &nonce, (*[32]byte)(clientPub), (*[32]byte)(s.serverPriv)),
+			Capublickey: s.marshalledCAPublicKey,
+			// Always trust the CA for now since the DNS gets resolved behind tunnel and therefore the client doesnt know
+			// which to trust
+			Cahosts:     []string{"*"},
```

Comment:
This should also return the private key for the user, correct? I assume that's coming in the next PRs?

Reply:
Yup, need #52 to generate the Keys from the ca.Signer